src/main/java/javaslang/collection/Map.java

```diff
@@ -39,6 +39,11 @@ default V apply(K key) {
 
     <U, W> Map<U, W> map(BiFunction<? super K, ? super V, ? extends Tuple2<? extends U, ? extends W>> mapper);
 
+    default <U> Seq<U> flatten(BiFunction<K, V, ? extends U> mapper) {
```

Comment:
Great! Love this method.

Please rename it to `traverse` (same functionality as `Tree.traverse()`) and move it behind `int size();`.

Thx!